api/azure/api.go

```diff
@@ -122,10 +138,13 @@ func (a apiImpl) FetchAzureArtifact(artifact BuildArtifact, w io.Writer) error {
 				log.Errorf("Failed to close gzip writer")
 				return err
 			}
-			return nil
+			extracted++
 		}
 	}
-	return fmt.Errorf("File %s not found in zip", reportPath)
+	if extracted < 1 {
+		return errors.New("No wpt_report.json files found in zip")
```

Comment:
```suggestion
		return errors.New(`No "wpt_report.*\.json" files found in zip`)
```

to make it clear that we allow arbitrary suffixes after `wpt_report`.